ipaclient/install/client.py

```diff
@@ -1364,7 +1364,6 @@ def do_nsupdate(update_txt):
 
     with open(UPDATE_FILE, "w") as f:
         f.write(update_txt)
-        ipautil.flush_sync(f)
```

Comment:
Doesn't this ensure that the update file is written to disk and not cached so that it is properly passed to nsupdate?

Reply:
The flush is unnecessary because the file descriptor will be closed after the compound statement.
The fsync is unnecessary too: it would be necessary to ensure the file's content after a hard reboot, but we do not care about that one ; if the box is forcefully rebooted during this stage, the enrollment will have failed anyway.